modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PolicyDetails/index.tsx

```diff
@@ -19,10 +19,10 @@ import {
 	getPolicyByExternalReferenceCode,
 } from '../../../../common/services';
 import {getQuotesById} from '../../../../common/services/Quote';
+import Activities from '../components/Activities/index';
```

Comment:
```suggestion
import Activities from '../components/Activities';
```

Reply:
/index is not needed to be imported, the folder name already looks for it

Reply:
Done. Thx!